ocaml/libs/http-lib/http_svr.ml

```diff
@@ -166,6 +166,10 @@ let response_forbidden ?req s =
   in
   response_error_html ?version s "403" "Forbidden" [] body
 
+let response_forbidden_with_body ?req s body =
+  let version = Option.map get_return_version req in
+  response_error_html ?version s "403" "Forbidden" [] body
```

Comment:
Do we want 403, not 503 as before?

Reply:
I think it is more like a service side issue. It's server that is applying the restriction.
How about 503 + a specific error message?